odd-collector/odd_collector/adapters/ckan/client.py

```diff
@@ -47,16 +52,51 @@ async def _post_request(self, url: str, payload: dict = None) -> dict:
         ) as session:
             try:
                 async with session.post(url, json=payload) as resp:
-                    result = await resp.json()
-                    logger.debug(f"Result of request {url} is {result}")
+                    if resp.status != 200:
```

Comment:
We have static method `is_response_successful`. Can we use this instead? Because 200 is not the only success http code. What if we get 201, 202 etc?

Reply:
In this case when we are processing "/api/action/datastore_info" endpoint we can get not "default" for CKAN API response without any "success" argument in it, so `is_response_successful` not applies to it. But I increased possible amount of successful status codes. Also maybe in future it is worth to do all checks based on status code, was not diving in it.